### PR TITLE
:recycle: small improvements

### PR DESCRIPTION
**Pull request by Sigmanificient**
- optimized import
- removed useless code
- pep8

### CODE REVIEW

File: pincer/middleware/interaction_create.py [hunk 1]
```diff
@@ -3,8 +3,8 @@
 
 from __future__ import annotations
 
-from contextlib import suppress
 import logging
+from contextlib import suppress
 from inspect import isasyncgenfunction, _empty
 from typing import Dict, Any
 from typing import TYPE_CHECKING
```
Comment (Lunarmagpie) [inline pincer/middleware/interaction_create.py:10]: Shouldn't logging be moved under inspect so its alphabetical?

File: pincer/middleware/stage_instance_update.py [hunk 2]
```diff
@@ -8,8 +8,8 @@
 from typing import TYPE_CHECKING
 
 from ..objects import StageInstance
-from ..utils.conversion import construct_client_dict
 from ..utils import Coro, replace
+from ..utils.conversion import construct_client_dict
```
Comment (Lunarmagpie) [inline pincer/middleware/stage_instance_update.py:12]: Can this be combined?
```python
from ..utils import Coro, replace, import construct_client_dict
```

File: pincer/objects/guild/channel.py [hunk 3]
```diff
@@ -6,8 +6,8 @@
 from asyncio import sleep, ensure_future
 from dataclasses import dataclass
 from enum import IntEnum
-from urllib.parse import urlencode
 from typing import AsyncIterator, overload, TYPE_CHECKING
+from urllib.parse import urlencode
```
Comment (Lunarmagpie) [inline pincer/objects/guild/channel.py:10]: I wouldn't recommend touching this. There will be a merge conflict with #343.

File: pincer/objects/guild/guild.py [hunk 4]
```diff
@@ -455,16 +455,16 @@ async def get_member(self, _id: int) -> GuildMember:
...
+            self,
+            *,
+            _id: int,
+            nick: Optional[str] = None,
+            roles: Optional[List[Snowflake]] = None,
+            mute: Optional[bool] = None,
+            deaf: Optional[bool] = None,
+            channel_id: Optional[Snowflake] = None,
+            reason: Optional[str] = None,
+            communication_disabled_until: Optional[Timestamp] = MISSING,
```
Comment (Lunarmagpie) [inline pincer/objects/guild/guild.py:467]: I think the indentation level is supposed to be one lower in this entire file.

File: pincer/utils/types.py [hunk 5]
```diff
@@ -4,7 +4,7 @@
 
 from sys import modules
 from typing import (
-    TYPE_CHECKING, TypeVar, Callable, Coroutine, Any, Union, Optional
+    TypeVar, Callable, Coroutine, Any, Union, Optional
 )
```
Comment (Lunarmagpie) [inline pincer/utils/types.py:8]: I think this can be one line

**Review by Lunarmagpie — CHANGES_REQUESTED** [review]

File: pincer/middleware/interaction_create.py (hunk 1 above)
Comment (Sigmanificient) [reply pincer/middleware/interaction_create.py:10]: I have always seperated `import` and `from` when sorting my imports

File: pincer/middleware/stage_instance_update.py (hunk 2 above)
Comment (Sigmanificient) [reply pincer/middleware/stage_instance_update.py:12]: maybe ye, but i dont wanna risk ciruclar import, need to be tested first

File: pincer/utils/api_object.py [hunk 6]
```diff
@@ -14,7 +14,6 @@
 )
 
 from pincer.utils.conversion import construct_client_dict
```
Comment (trag1c) [inline pincer/utils/api_object.py:16]: Shouldn't this be `.conversion` instead of `pincer.utils.conversion`?

**Review by trag1c — APPROVED** [review]

File: pincer/middleware/stage_instance_update.py (hunk 2 above)
Comment (Lunarmagpie) [reply pincer/middleware/stage_instance_update.py:12]: test it :popcorn:

File: pincer/middleware/interaction_create.py (hunk 1 above)
Comment (Lunarmagpie) [reply pincer/middleware/interaction_create.py:10]: yeah, that seems reasonable.

**Pull request merged**